src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -1,20 +1,11 @@
 echo off
 setlocal
 
-echo BridgeKeepRunning=%BridgeKeepRunning%
-if '%BridgeKeepRunning%' neq 'true' (
-    echo Releasing Bridge resources and stopping it if running locally.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
-    call Bridge.exe -stopIfLocal -reset %*
-    popd
-) else (
-    echo Releasing Bridge resources but leaving it running.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -reset %* ...
-    call Bridge.exe -reset %*
-    popd
-    echo The Bridge was left running because BridgeKeepRunning is true
-)
+echo Releasing Bridge resources and stopping it if running locally.
+pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+echo Invoking Bridge.exe -stopIfAutoStart -reset %* ...
+call Bridge.exe -stopIfAutoStart -reset %*
```

Comment:
The unspoken design goal here is that I can start the Bridge manually for debugging, and no test script is permitted to shut it down.  Or another process can be managing the lifetime of the Bridge and does not want test scripts to stop it. It is literally a question of who owns the lifetime of the Bridge. To respond to @zhenlen, the Bridge should remember how it was started, because it will condition under what circumstances it should be stopped.  There is no persistence mechanism between test runs or machines other than the Bridge itself.

I think I will refactor to a single boolean flag "BridgeAutoStart" that is set from any test script that causes the Bridge to start.  It will make the new command "-stopIfAutoStart" make more sense.